daft/dataframe/dataframe.py

```diff
@@ -2118,6 +2118,17 @@ def mean(self, *cols: ColumnInputType) -> "DataFrame":
         """
         return self._apply_agg_fn(Expression.mean, cols)
 
+    @DataframePublicAPI
+    def stddev(self, *cols: ColumnInputType) -> "DataFrame":
+        """Performs a global standard deviation on the DataFrame
+
```

Comment:
Add example? (and add stddev to docs as well)

Reply:
Ya good point. Will update and push (in this PR).